docs/reference/api/platform/Debug.md

```diff
@@ -1,11 +1,36 @@
 ## Debug
 
-[Add description here.]
+Mbed OS provides a set of debug functions that you can use to output debug messages to `STDIO` at runtime. `mbed_debug.h` declares these functions, which are available only in debug builds.
 
-### Debug class reference
+### Debug functions reference
 
-[Embed class reference here. If you've never done this before, please ask your editor.]
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/group__platform__debug.html)
+
+### Debug functions usage
```

Comment:
I like the content in this section, but this section breaks the standardization with the other APIs. Could we make it a second example or move some of the content up to the description area?

Reply:
I see, yes, we can move it to description area.